cmd/podman/common/create.go

```diff
@@ -140,7 +140,9 @@ func DefineCreateFlags(cmd *cobra.Command, cf *entities.ContainerCreateOptions,
 		if !registry.IsRemote() {
 			createFlags.BoolVar(
 				&cf.EnvHost,
-				"env-host", false, "Use all current host environment variables in container",
+				"env-host",
+				podmanConfig.ContainersConfDefaultsRO.Containers.EnvHost,
```

Comment:
This is not the the best (I would even say correct) place to set defaults.

Mainly this will bypass the server defaults, so it will not take effect for users that use the API.

Reply:
The change is not for remote. So it does not bypass server, since it does not work with remote.

The difficulty difference between --remote and --remote=false is when you execute `podman CMD --help` cobra is listing the defaults.  In the local case we can figure out the current defaults from containers.conf, in the --remote case we can not figure these defaults so we are forced to just list what we got.

Reply:
@rhatdan what Paul pointed at is that `podman system service` is not loading these defaults, so clients (e.g., podman-py) will not notice any changes when they configure containers.conf.  That's a fart in the REST API and how we use containers.conf

But getting this done (IMO) is outside of the context of this PR and probably quite some work.

Reply:
Agreed.